test/unit/BabelPluginExportToFunctionSpec.js

```diff
@@ -82,7 +115,9 @@ test('BabelPluginExportToFunction should throw on not enough arguments', (t) =>
         'exports.default = rule;'
     ].join('\n');
 
-    t.throws(() => babel.transform(code, { plugins: [ exportToFunction ] }));
+    const error = t.throws(() => babel.transform(code, { plugins: [ exportToFunction ] }));
+
+    t.is(error.message, 'unknown: auth0-bundler is missing default export in module');
```

Comment:
Why is this still expected to be an error?
I also don’t understand what the error message has to do with "not enough arguments".